README.rst

```diff
@@ -0,0 +1,54 @@
+..
+    This file is part of Invenio.
+    Copyright (C) 2015 CERN.
+
+    Invenio is free software; you can redistribute it
+    and/or modify it under the terms of the GNU General Public License as
+    published by the Free Software Foundation; either version 2 of the
+    License, or (at your option) any later version.
+
+    Invenio is distributed in the hope that it will be
+    useful, but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+    General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with Invenio; if not, write to the
+    Free Software Foundation, Inc., 59 Temple Place, Suite 330, Boston,
+    MA 02111-1307, USA.
+
+    In applying this license, CERN does not
+    waive the privileges and immunities granted to it by virtue of its status
+    as an Intergovernmental Organization or submit itself to any jurisdiction.
+
+======================================
+Invenio Schema
+======================================
+
+.. image:: https://img.shields.io/travis/inveniosoftware/invenio-schema.svg
+        :target: https://travis-ci.org/inveniosoftware/invenio-schema
+
+.. image:: https://img.shields.io/coveralls/inveniosoftware/invenio-schema.svg
+        :target: https://coveralls.io/r/inveniosoftware/invenio-schema
+
+.. image:: https://img.shields.io/github/tag/inveniosoftware/invenio-schema.svg
+        :target: https://github.com/inveniosoftware/invenio-schema/releases
+
+.. image:: https://img.shields.io/pypi/dm/invenio-schema.svg
+        :target: https://pypi.python.org/pypi/invenio-schema
+
+.. image:: https://img.shields.io/github/license/inveniosoftware/invenio-schema.svg
+        :target: https://github.com/inveniosoftware/invenio-schema/blob/master/LICENSE
+
+
+Invenio Schema Registry
```

Comment:
Too many different names in this PR, without consistency between Python package name and textual product full name? 

- (1) `invenio-schema` looks like this repository would be containing only a single schema... not even mentioning JSON.  The textual name equivalent would be Invenio-Schema, which sounds incomplete and/or misleading.

- (2) `invenio-jsonschemata` is how this repository is currently called, so that would be the package name (with underscore).  Its full name equivalent to advertise would be "Invenio-JSONSchemata".  (Perhaps with hyphen between json and schema for better readability? `invenio-json-schemata` and "Invenio-JSON-Schemata"?) 

- (3) "Invenio Schema Registry" does not sound bad either, but then the package should be called `invenio-schema-registry` for consistency. Again, probably with JSON added both in title and in repository name, `invenio-jsonschema-registry` and "Invenio-JSONSchema-Registry".  

Please see how we did consistent naming in `invenio-query-parser`.  You can emulate that.

I'd go for the current name `invenio-jsonschemata`, unless people hate the canonical plural form stemming from its Latin roots; in which case `invenio-jsonschemas` could be an (uglier :) possibility, or else more neutral `invenio-jsonschema-registry` which does not sound too bad either.